testsuite/features/support/twopence_init.rb

```diff
@@ -223,6 +225,9 @@ def file_inject(node, local_file, remote_file)
 # Other global variables
 $product = product
 $pxeboot_mac = ENV['PXEBOOT_MAC']
+$sle11sp3_terminal = ENV['SLE11SP3_TERMINAL']
+$sle12sp5_terminal = ENV['SLE12SP5_TERMINAL']
+$sle15sp3_terminal = ENV['SLE15SP3_TERMINAL']
```

Comment:
Can I ask for a rename of this variable?
I would prefer xxx_terminal_mac so we can better distinguish these ones from the variables containing twopence objects.

Reply:
ok. It would make sense also if we introduce `sle11sp3_terminal_image`.